tests/runner/testHarnessMain.js

```diff
@@ -0,0 +1,50 @@
+
+require(['testHarnessConfig'], function() {
+    require([
+        'bouncefix',
+        '$',
+        'velocity',
+        'modal-center',
+        'sheet-bottom',
+        'sheet-left',
+        'sheet-right',
+        'sheet-top',
+        'plugin',
+        'shade',
+        'deckard',
+        'lockup',
+        'pinny'
+    ],
+        function(
+            bounceFix, 
+            $,
+            velocity, 
+            modalCenter, 
+            sheetBottom, 
+            sheetLeft, 
+            sheetRight, 
+            sheetTop, 
+            plugin, 
+            shade, 
+            deckard, 
+            lockup, 
+            pinny) {
```

Comment:
Let's try and dynamically bind all the `arguments` to a global `dependencies` object. That will make this uber dynamic!